pkg/user/builder.go

```diff
@@ -92,3 +94,11 @@ func (b *Builder) Auths(auths []Auth) *Builder {
 	b.u.auths = append([]Auth{}, auths...)
 	return b
 }
+
+func (b *Builder) PasswordReset(token string, createdAt time.Time) *Builder {
```

Comment:
```suggestion
func (b *Builder) PasswordReset(pr *PasswordReset) *Builder {
```

Accepting PasswordReset struct directly make codes simple

Reply:
@rot1024 
I tried this one it cause an import loop!